loadtest/bulkload.go

```diff
@@ -512,13 +512,30 @@ func GenerateBulkloadFile(config *LoadtestEnviromentConfig) GenerateBulkloadFile
 		}
 	}
 
+	randString := func() string {
+		i := r.Uint64()
+		// pick a random base (between 2 and 36), ensuring a non-uniform distribution
+		base := r.Intn(34) + 2
+		return strconv.FormatUint(i, base)
+	}
+
 	for userNum := 0; userNum < config.NumUsers; userNum++ {
-		users = append(users, UserImportData{
-			Username: "user" + strconv.Itoa(userNum),
+		user := UserImportData{
+			Username: randString(),
```

Comment:
Might we adopt something like the strategy in `makeChannelName` instead, with the change that some usernames have common prefixes (as they would in real life)? A numeric username probably isn't that representative unto itself either.

Reply:
Ah I figured there must be a helper function for random strings somewhere but wasn't finding anything (although I didn't look that hard 😛). Seems like fake.Characters(r.Intn(5)+3) may be good.

Reply:
Sounds good -- just keep in mind that the `username` must be unique -- and we often test with 20,000+ users -- so it may be worth embedding the `userNum` into the username too to avoid a future issue.

Reply:
Yep actually I ran into that while testing this fix. The fake package doesn't do a great job since it's pulling from lists of about 200 words. I just appended the userNum similarly to makeChannelName